core/types/gen_header_rlp.go

```diff
@@ -1,85 +0,0 @@
-// Code generated by rlpgen. DO NOT EDIT.
```

Comment:
Because we define our own EncodeRLP in `celo_block.go` that handles pre and post gingerbread rlp encodings.

Reply:
Sure, but there it's just `rlp.Encode(w, &encodedHeader)` to encode it. If it's that easy, wthen why did all of this code exist? Will this have an impact on performance?

Reply:
Yeah it was performance see here - https://github.com/ethereum-optimism/op-geth/commit/d6f49bf764c48e9415b168e701b6dc26566a675c

So we could probably add encoding like this back in for the 2 types we support. But I'd rather leave that for later since right now performance is not an issue.

Reply:
Added https://github.com/celo-org/op-geth/issues/145